tests/tck/features/lookup/EdgeIndexFullScan.feature

```diff
@@ -233,55 +324,18 @@ Feature: Lookup edge index full scan
     Then a SemanticError should be raised at runtime: Column type error : col1_str
     When profiling query:
       """
-      LOOKUP ON edge_1 WHERE edge_1.col1_str NOT STARTS WITH toUpper("r") YIELD edge_1.col1_str
+      LOOKUP ON edge_1 WHERE edge_1.col1_str NOT STARTS WITH "R" YIELD edge_1.col1_str
       """
-    Then the result should be, in any order:
-      | SrcVID | DstVID | Ranking | edge_1.col1_str |
-      | "103"  | "101"  | 0       | "Blue"          |
-      | "102"  | "103"  | 0       | "Yellow"        |
-    And the execution plan should be:
-      | id | name              | dependencies | operator info                                            |
-      | 3  | Project           | 2            |                                                          |
-      | 2  | Filter            | 4            | {"condition": "(edge_1.col1_str NOT STARTS WITH \"R\")"} |
-      | 4  | EdgeIndexFullScan | 0            |                                                          |
-      | 0  | Start             |              |                                                          |
+    Then a SemanticError should be raised at runtime: Expression (edge_1.col1_str NOT STARTS WITH "R") is not supported, please use full-text index as an optimal solution
 
   Scenario: Edge with relational ENDS/NOT ENDS WITH filter
-    When profiling query:
-      """
-      LOOKUP ON edge_1 WHERE edge_1.col1_str ENDS WITH toLower("E") YIELD edge_1.col1_str
-      """
-    Then the result should be, in any order:
-      | SrcVID | DstVID | Ranking | edge_1.col1_str |
-      | "103"  | "101"  | 0       | "Blue"          |
-    And the execution plan should be:
-      | id | name              | dependencies | operator info                                      |
-      | 3  | Project           | 2            |                                                    |
-      | 2  | Filter            | 4            | {"condition": "(edge_1.col1_str ENDS WITH \"e\")"} |
-      | 4  | EdgeIndexFullScan | 0            |                                                    |
-      | 0  | Start             |              |                                                    |
     When executing query:
       """
       LOOKUP ON edge_1 WHERE edge_1.col1_str ENDS WITH "ABC" YIELD edge_1.col1_str
       """
-    Then the result should be, in any order:
-      | SrcVID | DstVID | Ranking | edge_1.col1_str |
+    Then a SemanticError should be raised at runtime: Expression (edge_1.col1_str ENDS WITH "ABC") is not supported, please use full-text index as an optimal solution
```

Comment:
For this, maybe we could schedule index scan and full-text search automatically. Of course not in current PR.